beacon-chain/blockchain/forkchoice/log.go

```diff
@@ -0,0 +1,27 @@
+package forkchoice
+
+import (
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.WithField("prefix", "forkchoice")
+
+// logs epoch related data during epoch boundary.
+func logEpochData(beaconState *pb.BeaconState) {
+	log.WithField(
+		"previousJustifiedEpoch", beaconState.PreviousJustifiedCheckpoint.Epoch,
+	).Info("Previous justified epoch")
```

Comment:
Instead of 5 log statements, can this be one log with 5 fields? It would be easier to parse if there was something ingesting JSON logs